cirq/contrib/quirk/quirk_gate.py

```diff
@@ -116,39 +124,38 @@ def _gate_to_quirk_op(gate: ops.Gate) -> Optional[QuirkOp]:
     return None
 
 
-def x_to_known(gate: ops.XPowGate) -> Optional[QuirkOp]:
-    e = angle_to_exponent_key(gate.exponent)
-    if e is None:
-        return None
-    return QuirkOp('X' + e)
+def xyz_to_known(axis: str, gate: ops.EigenGate) -> QuirkOp:
+    d = axis.lower()
+    u = axis.upper()
 
+    if gate.global_shift == -0.5:
+        return QuirkOp({'id': 'R{d}}ft', 'arg': f'{gate.exponent:.4f}pi'})
```

Comment:
nit: By comparing to l.138, it seems you may have meant ^ instead of second }, i.e. 'R{d}^ft'.

What's "ft"?

Reply:
Removed the extra }.

ft is an abbreviation of "function of t".